.github/workflows/tests.yml

```diff
@@ -0,0 +1,48 @@
+name: Tests
+
+on:
+  push:
+    paths-ignore:
+      - 'Docs/**'
+  pull_request:
+    paths-ignore:
+      - 'Docs/**'
+
+jobs:
+  tests:
+    strategy:
+      fail-fast: true
+      matrix:
+        php: ["7.3", "7.4", "8.0", "8.1", "8.2", "8.3"]
+
+    name: "PHP ${{ matrix.php }}"
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: "${{ matrix.php }}"
+          coverage: none
```

Comment:
I'm not really sure how to implement code coverage reporting with GitHub actions so I left it out for now. I hope that's okay.